notes/returned-mail.vbs

```diff
@@ -336,65 +357,73 @@ ELSE
 		EMReadScreen error_message,  74, 24, 02 'for script_run_lowdown-reading for messages that might be missed if they are not inhibiting'
 		error_message = trim(error_message)
     END IF 'if snap or cash are true'
-	'we cannot close HC currently but this is the place for that handling'
-END IF 'forwarding address provided
-
-'----------------------------------------------------------------------------------------------------TIKL
-'Call create_TIKL(TIKL_text, num_of_days, date_to_start, ten_day_adjust, TIKL_note_text)
-IF ADDR_actions <> "no response received" THEN Call create_TIKL("Returned mail rec'd contact from the client should have occurred regarding address change. If no response-verbal or written, please take appropriate action.", 10, date, True, TIKL_note_text)
+    'we cannot close HC currently but this is the place for that handling'
+END IF 'no response received '
+    '----------------------------------------------------------------------------------------------------TIKL
+IF ADDR_actions = "forwarding address provided" or ADDR_actions = "no forwarding address provided" THEN
+    'Call create_TIKL(TIKL_text, num_of_days, date_to_start, ten_day_adjust, TIKL_note_text)
+    Call create_TIKL("Returned mail rec'd contact from the resident should have occurred regarding address change. If no response-verbal or written, please take appropriate action.", 10, date, True, TIKL_note_text)
```

Comment:
1st sentence is a run on sentence. Add period.

Reply:
Adjusted TIKL to make a complete sentence